nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Updater/BindingRedirectManager.cs

```diff
@@ -154,42 +154,6 @@ static bool IsConfigFile(IXmlElementSyntax element)
             return (element.Name == "None" && string.Equals(path, "app.config", StringComparison.OrdinalIgnoreCase))
                    || (element.Name == "Content" && string.Equals(path, "web.config", StringComparison.OrdinalIgnoreCase));
         }
-
-        static string GetConfigFileName(XmlDocumentSyntax document)
```

Comment:
These functions were not referenced, removed to avoid confusion. Let me know if you want them restored.